packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -385,14 +405,50 @@ export class Install {
                 console.error(`host.${hostname} Cannot install ${npmUrl}: ${result.exitCode}`);
                 return this.processExit(EXIT_CODES.CANNOT_INSTALL_NPM_PACKET);
             }
-            // create file that indicates, that npm was called there
+            // create file that indicates that npm was called there
             fs.writeFileSync(path.join(installDir, 'iob_npm.done'), ' ');
             // command succeeded
             return { _url: npmUrl, installDir: path.dirname(installDir) };
         } else {
-            console.error(result.stderr);
-            console.error(`host.${hostname} Cannot install ${npmUrl}: ${result.exitCode}`);
-            return this.processExit(EXIT_CODES.CANNOT_INSTALL_NPM_PACKET);
+            // if we see ENOTEMPTY error
+            if (!isRetry && result.stderr.includes('ENOTEMPTY')) {
+                // try to manually remove the directory and start installation again
+                // detect node_modules folder
+                const pack = await tools.detectPackageManagerWithFallback();
+                // extract adapter name
```

Comment:
juist call variable `adapterName` then it is self-explanatory and we not need the comment

Reply:
Done